examples/sssp/sssp.cu

```diff
@@ -1,6 +1,5 @@
 #include <cstdlib>  // EXIT_SUCCESS
 
-#include <gunrock/applications/runner.hxx>
 #include <gunrock/applications/sssp/sssp_implementation.hxx>
```

Comment:
I think we should use `sssp.hxx` as the `#include` for `sssp`, it stays with the standard of keeping a header file along with a c/c++ driver.

Reply:
you mean rename `sssp_implementation.hxx` to `sssp.hxx`?

Reply:
Yes